### ISSUE TITLE
useComputed outside useObserver is uncached

### ISSUE THREAD

**Issue by xaviergonz** [mobxjs/mobx-react-lite#106]
since useComputed returns a plain value, this means that if it is used outside a useObserver block it will never cache, and therefore just be a more expensive "always calculate this"

e.g.
```
const sum = useComputed(() => 1+2); // this will return the plain value 3, and since get() is called outside observation it is not cache
useObserver(() => {
  // if we use sum here it will just use the plain value 3, which cannot be tracked
})
```

**Comment by xaviergonz** [comment]
One way to fix it I guess would be to return a
```
{
  get current() { return computed.get() } // readonly
}
```

and then use it like
```
const sum = useComputed(() => 1+2)
useObserver(() => {
  // if we use sum.current here it will actually call get, which can be tracked
})
```

**Comment by FredyC** [comment]
I wouldn't bother really since we are considering removal of it ... #94

**Comment by FredyC** [comment]
Closing since we have indeed deprecated `useComputed` in favor of `useLocalStore`. Released in 1.3.0
